integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/automaticindexing/association/bytype/onetoone/ownedbycontained/AutomaticIndexingOneToOneOwnedByContainedLazyOnContainedSideIT.java

```diff
@@ -89,6 +90,11 @@ protected boolean isAssociationLazyOnContainingSide() {
 	public void setup(OrmSetupHelper.SetupContext setupContext,
 			ReusableOrmSetupHolder.DataClearConfig dataClearConfig) {
 		super.setup( setupContext, dataClearConfig );
+		// Avoid problems with deep chains of eager associations in ORM 6
+		// See https://github.com/hibernate/hibernate-orm/blob/main/migration-guide.adoc#fetch-behaviour-change
```

Comment:
I wanted to learn more about the changes in fetching, and the link looks broken now 😢.
The somewhat related to the fetching section in the 6.0 guide - https://github.com/hibernate/hibernate-orm/blob/6.0/migration-guide.adoc#fetch-circularity-determination is a bit confusing (unfinished or repeating sentences) and sadly I'm not with enough knowledge to update it 😞

Reply:
Yes I guess that's the right section, I'll update the links. As to updating the migration guide, maybe ping Christian about that?

Reply:
thanks, will do 😃

Reply:
Anyway the gist of it is "if you have multiple circular eager associations, ORM will now (correctly) follow them and your queries will become exponentially larger as the depth increases, so you'd better limit the depth"